### ISSUE TITLE
not all repos for an organization are shown

### ISSUE THREAD

**Issue by raboof** [slapperwan/gh4a#235]
could only be an issue with private repos, not sure. Some private repos are visible though...

**Comment by raboof** [comment]
When I jump to a project via a link this works fine, but it doesnt show up when enumerating all projects for a project

**Comment by nflbeezy** [comment]
+1

**Comment by alorma** [comment]
Github recently changed how private repos are shown in the apps.

To make it appear, first check your settings -> applications, and check if the application is allowed to access that organization

**Comment by raboof** [comment]
@alorma I'm not sure what you mean - on https://github.com/settings/applications gh4a doesn't show up as an application, and access is not granted on a per-application basis.

**Comment by maniac103** [comment]
Right, Octodroid uses the authorizations API to create the OAuth tokens instead of the full OAuth flow.

**Comment by alorma** [comment]
Yes @raboof, in that page, if you select the application (octodroid, or gh4a), you can grant acces

**Comment by alorma** [comment]
@maniac103 with auth api, users don't need to grant access to the applications?

**Comment by maniac103** [comment]
@alorma Correct. This is the API we use: https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization
The web flow is a bit cumbersome from a UX perspective IMHO, and the setup for different build variants to store the client secret etc. also creates more work ;)
